__tests__/app.test.js

```diff
@@ -170,3 +170,39 @@ describe("GET: /api/articles/:article_id/comments", () => {
       });
   });
 });
+
+describe("POST: /api/articles/:article_id/comments", () => {
+  test("200: adds a comment for an article and responds with the posted comment", () => {
+    const newComment = {
+      username: "icellusedkars",
+      body: "Hit me up if you wanna sell your wheelz",
+    };
+    return request(app)
+      .post("/api/articles/9/comments")
+      .send(newComment)
+      .expect(201)
+      .then(({ body: { comment } }) => {
+        expect(comment).toEqual(
+          expect.objectContaining({
+            comment_id: expect.any(Number),
+            body: "Hit me up if you wanna sell your wheelz",
+            article_id: 9,
+            author: "icellusedkars",
+            votes: 0,
+            created_at: expect.any(String),
+          })
+        );
+      });
+  });
+  test("400: responds with an appropriate status and error message when provided without a body", () => {
+    return request(app)
+      .post("/api/articles/9/comments")
+      .send({
+        username: "icellusedkars",
+      })
+      .expect(400)
+      .then(({ body: { msg } }) => {
+        expect(msg).toBe("Bad Request");
+      });
+  });
+});
```

Comment:
Okay good testing so far but there's a few more errors you can add! 

What if the article_id is not found? 
What if the article_id is invalid? 
What if the username does not exist?